### DISCUSSION TITLE
Product Feedback for Support Page in German

### DISCUSSION THREAD

**Discussion by GunnarBernsteinHH** [community/community#83649] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have a browser set to German. As a result I get presented a German Page when going to https://support.github.com/. 
I rather be on the English Support Page, because a search for "remove sensible data" does not yield any result.
Since the Feedback link is redirected here, I assume the Feedback should go here.

I always appreciate it when a button to switch the language or region is somewhere on the page.

**Comment by developertanvir2019** [comment]
following
